src/main/java/hudson/scm/SubversionSCM.java

```diff
@@ -1354,9 +1353,8 @@ else if (project.getLastBuild()!=null) {
             }
             if (project instanceof AbstractProject && repositoryLocationsNoLongerExist(lastCompletedBuild, listener, env)) {
                 // Disable this project, see HUDSON-763
-                listener.getLogger().println(
-                        Messages.SubversionSCM_pollChanges_locationsNoLongerExist(project));
-                disableProject((AbstractProject) project, listener);              
+                listener.getLogger().println(Messages.SubversionSCM_pollChanges_locationsNoLongerExist(project));
```

Comment:
@daniel-beck I agree. I'm trying to apply a coding style and I do modifications in each PRs.